Ju5ts0cRypt0 3 projects

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "project": "Cardano Creatures",
+        "policies": [
+            "f976d018b11cd9e7168c199344cbbd65d6b7e4dad81bb40908477714",
+            "b6831985e6385d9b20d0914e3a1864e0d8bf288072871d2ded1b9e18"
+        ]
+    },
+    {
+        "project": "Cosmic Cannabis",
+        "policies": [
+            "f1332030dc83f8dd4765819c34c65e41269e93c4895e4385f6342da2"
+        ]
+    }
```

Comment:
You need a comma here after the closing curly bracket.

Reply:
Do I need to resubmit? Sorry for the ignorance here.

Reply:
I'm nobody here. I just noticed an issue and thought I'd point it out.

But I don't think you will need a new PR. You already updated the JSON so you good.